easybuild/easyblocks/c/cp2k.py

```diff
@@ -729,8 +729,19 @@ def test_step(self):
             # location of do_regtest script
             cfg_fn = 'cp2k_regtest.cfg'
 
-            regtest_script = os.path.join(self.cfg['start_dir'], 'tools', 'regtesting', 'do_regtest')
-            regtest_cmd = [regtest_script, '-nobuild', '-config', cfg_fn]
+            if LooseVersion(self.version) > LooseVersion('2024.0'):
+                regtest_script = os.path.join(self.cfg['start_dir'], 'tests', 'do_regtest.py')
+                regtest_cmd = [
+                    'python',
+                    regtest_script,
+                    '--maxtasks 1 --mpiranks 1 --ompthreads 1',
```

Comment:
Running tests single-core as default makes sense, partially because we know these tests can be quite memory hungry, but it would be nice to provide custom easyconfig parameters so people can tweak these values if they have enough memory in the easyconfig file (or through a hook), like:
```python
tests_maxtasks = 2
tests_mpiranks = 4
tests_ompthreads = 2
```

and same goes for the `--maxerrors` and `--timeout` options.

I would use a very high value for default for `maxerrors`, to always get a full result of the test suite, like 10k (`10000`). And likewise for `timeout` (`1000` makes sense there I think)